[2024-02-19T04:23:58.414+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: final_project.Run_data_quality_checks scheduled__2024-02-19T03:00:00+00:00 [queued]>
[2024-02-19T04:23:58.419+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: final_project.Run_data_quality_checks scheduled__2024-02-19T03:00:00+00:00 [queued]>
[2024-02-19T04:23:58.419+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 4
[2024-02-19T04:23:58.425+0000] {taskinstance.py:2192} INFO - Executing <Task(DataQualityOperator): Run_data_quality_checks> on 2024-02-19 03:00:00+00:00
[2024-02-19T04:23:58.428+0000] {standard_task_runner.py:60} INFO - Started process 29075 to run task
[2024-02-19T04:23:58.430+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'final_project', 'Run_data_quality_checks', 'scheduled__2024-02-19T03:00:00+00:00', '--job-id', '129', '--raw', '--subdir', 'DAGS_FOLDER/final_project.py', '--cfg-path', '/tmp/tmp0s_umi8b']
[2024-02-19T04:23:58.433+0000] {standard_task_runner.py:88} INFO - Job 129: Subtask Run_data_quality_checks
[2024-02-19T04:23:58.459+0000] {task_command.py:423} INFO - Running <TaskInstance: final_project.Run_data_quality_checks scheduled__2024-02-19T03:00:00+00:00 [running]> on host 6b28116ad14a
[2024-02-19T04:23:58.495+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='udacity' AIRFLOW_CTX_DAG_ID='final_project' AIRFLOW_CTX_TASK_ID='Run_data_quality_checks' AIRFLOW_CTX_EXECUTION_DATE='2024-02-19T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-19T03:00:00+00:00'
[2024-02-19T04:23:58.500+0000] {base.py:83} INFO - Using connection ID 'aws_credentials' for task execution.
[2024-02-19T04:23:58.500+0000] {connection_wrapper.py:378} INFO - AWS Connection (conn_id='aws_credentials', conn_type='aws') credentials retrieved from login and password.
[2024-02-19T04:23:58.510+0000] {base.py:83} INFO - Using connection ID 'redshift' for task execution.
[2024-02-19T04:24:03.472+0000] {sql.py:432} INFO - Running statement: select count(*) from staging.staging_events, parameters: None
[2024-02-19T04:24:04.817+0000] {sql.py:441} INFO - Rows affected: 1
[2024-02-19T04:24:05.336+0000] {data_quality.py:36} INFO - Data quality on table staging.staging_events check passed with 8056 records
[2024-02-19T04:24:05.341+0000] {base.py:83} INFO - Using connection ID 'redshift' for task execution.
[2024-02-19T04:24:10.071+0000] {sql.py:432} INFO - Running statement: select count(*) from staging.staging_songs, parameters: None
[2024-02-19T04:24:11.300+0000] {sql.py:441} INFO - Rows affected: 1
[2024-02-19T04:24:11.856+0000] {data_quality.py:36} INFO - Data quality on table staging.staging_songs check passed with 14896 records
[2024-02-19T04:24:11.861+0000] {base.py:83} INFO - Using connection ID 'redshift' for task execution.
[2024-02-19T04:24:16.464+0000] {sql.py:432} INFO - Running statement: select count(*) from fact.fact_song_plays, parameters: None
[2024-02-19T04:24:17.899+0000] {sql.py:441} INFO - Rows affected: 1
[2024-02-19T04:24:18.499+0000] {data_quality.py:36} INFO - Data quality on table fact.fact_song_plays check passed with 6820 records
[2024-02-19T04:24:18.509+0000] {base.py:83} INFO - Using connection ID 'redshift' for task execution.
[2024-02-19T04:24:23.045+0000] {sql.py:432} INFO - Running statement: select count(*) from dim.dim_users, parameters: None
[2024-02-19T04:24:24.653+0000] {sql.py:441} INFO - Rows affected: 1
[2024-02-19T04:24:25.479+0000] {data_quality.py:36} INFO - Data quality on table dim.dim_users check passed with 104 records
[2024-02-19T04:24:25.491+0000] {base.py:83} INFO - Using connection ID 'redshift' for task execution.
[2024-02-19T04:24:31.002+0000] {sql.py:432} INFO - Running statement: select count(*) from dim.dim_songs, parameters: None
[2024-02-19T04:24:32.070+0000] {sql.py:441} INFO - Rows affected: 1
[2024-02-19T04:24:32.646+0000] {data_quality.py:36} INFO - Data quality on table dim.dim_songs check passed with 14896 records
[2024-02-19T04:24:32.668+0000] {base.py:83} INFO - Using connection ID 'redshift' for task execution.
[2024-02-19T04:24:38.204+0000] {sql.py:432} INFO - Running statement: select count(*) from dim.dim_artists, parameters: None
[2024-02-19T04:24:39.507+0000] {sql.py:441} INFO - Rows affected: 1
[2024-02-19T04:24:40.324+0000] {data_quality.py:36} INFO - Data quality on table dim.dim_artists check passed with 10025 records
[2024-02-19T04:24:40.355+0000] {base.py:83} INFO - Using connection ID 'redshift' for task execution.
[2024-02-19T04:24:45.443+0000] {sql.py:432} INFO - Running statement: select count(*) from dim.dim_time, parameters: None
[2024-02-19T04:24:46.668+0000] {sql.py:441} INFO - Rows affected: 1
[2024-02-19T04:24:47.301+0000] {data_quality.py:36} INFO - Data quality on table dim.dim_time check passed with 6820 records
[2024-02-19T04:24:47.307+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=final_project, task_id=Run_data_quality_checks, execution_date=20240219T030000, start_date=20240219T042358, end_date=20240219T042447
[2024-02-19T04:24:47.357+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-19T04:24:47.366+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
